eagle-stores/src/pages/Cart/components/CartCard.css

```diff
@@ -0,0 +1,126 @@
+.cart__container {
+    width: 100vw;
+    padding: 2rem 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.content__heading {
+    padding: 1rem 0;
+}
+
+.cart {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.cart__horizontal {
+    width: 30rem;
```

Comment:
Can give max or min-width for responsiveness, can add later on

Reply:
okay